src/opyrability.py

```diff
@@ -1760,7 +1761,7 @@ def implicit_map(model:             Callable[...,Union[float,np.ndarray]],
     jit:  bool True, optional
         JAX's Just-in-time compilation (JIT) of implicit function and its 
         respective multidimensional derivatives (Jacobians). JIT allows faster
-        computation of the implicit map. The default is 'True'.
+        computation of the implicit map. The default is True.
```

Comment:
The `implicit_map` is missing docstrings for `model`, `image_init`, and `domain_points`

Reply:
Also having `jit` as the boolean argument and the import (`from jax import jit, jacrev`) is ambiguous whether the code is properly functiong properly